site/src/comparison.rs

```diff
@@ -538,7 +506,7 @@ pub fn write_summary_table(
             Some(secondary.arithmetic_mean_of_improvements())
         }),
         if primary.is_empty() {
-            "N/A".to_string()
```

Comment:
I think "N/A" is more clear...

Reply:
What I don't like about it is that it produces visual clutter, it's harder to find the real changes among a see of `N/A`. Compare:

|            | mean[^1] | max | count[^2] |
|:----------:|:--------:|:---:|:---------:|
| Worse (primary) | 20.0%    | 20.0% | 1         |
| Worse (secondary) |          |     | 0         |
| Better (primary) | -50.0%   | -50.0% | 1         |
| Better (secondary) |          |     | 0         |
| All (primary) | -15.0%   | -50.0% | 2         |


|            | mean[^1] | max | count[^2] |
|:----------:|:--------:|:---:|:---------:|
| Worse (primary) | 20.0%    | 20.0% | 1         |
| Worse (secondary) |    N/A      |  N/A   | 0         |
| Better (primary) | -50.0%   | -50.0% | 1         |
| Better (secondary) |    N/A      |  N/A   | 0         |
| All (primary) | -15.0%   | -50.0% | 2         |

But same as with all the other things, I don't have any hard preference here.